modules/vaos/app/serializers/vaos/v0/va_appointments_serializer.rb

```diff
@@ -9,7 +9,7 @@ class VAAppointmentsSerializer
 
       set_type :va_appointments
 
-      attributes :start_date, :sta6aid, :clinic_id, :clinic_friendly_name, :facility_id, :community_care
+      attributes :start_date, :sta6aid, :clinic_id, :clinic_friendly_name, :facility_id, :char4, :community_care
```

Comment:
Now that I look at this, we might be missing `:phone_only`, too (`phoneOnly` from MAS). I can create a separate ticket for that, unless you want to add it in this PR?

Reply:
I'll include it here.